Resources/Textures/_RMC14/Objects/Weapons/Guns/LMGs/GSh.rsi/meta.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Based on parts from ml79a.rsi, minigun.rsi and the Colour sceme from the cm13 Sprite. Made by RavenCaat :3.",
```

Comment:
This also needs to include links to the files by commit, you can get those from GitHub if you weren't using RSIEdit